third-party/jemalloc/Makefile

```diff
@@ -15,7 +15,8 @@ CHPL_JEMALLOC_CFG_OPTIONS += --host=$(CHPL_MAKE_TARGET_PLATFORM)-unknown-linux-g
 endif
 
 CHPL_JEMALLOC_CFG_OPTIONS += --prefix=$(JEMALLOC_INSTALL_DIR) \
-			     --with-jemalloc-prefix=je_
+                             --with-jemalloc-prefix=je_ \
+                             --with-private-namespace=jeint_
```

Comment:
Agreed, this is small, but I think it should be split off into a separate PR. 

I'm also not a fan of jeint_, I think we should go with something with `chpl` init. My preference would be je_chpl_ or chpl_je_